server.js

```diff
@@ -1,27 +1,100 @@
-import cors from "cors";
-import express from "express";
-import mongoose from "mongoose";
+import cors from 'cors'
+import express from 'express'
+import mongoose from 'mongoose'
+import expressListEndpoints from 'express-list-endpoints'
+import dotenv from 'dotenv'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl);
-mongoose.Promise = Promise;
+dotenv.config()
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const mongoUrl = process.env.MONGO_URL || 'mongodb://localhost/project-mongo'
+mongoose.connect(mongoUrl)
+mongoose.Promise = Promise
+
+
+const port = process.env.PORT || 8080
+const app = express()
+
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+})
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+app.get('/', (req, res) => {
+  const endpoints = expressListEndpoints(app)
+  res.send(endpoints)
+})
+
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find()
+    .sort({ createdAt: 'desc' })
```

Comment:
Learning new things every time I do a code review. There are so many ways to do the descending thing. Love that about code and thanks for your inspiration to do it in another way. 😄